manifest/manifestlist/manifestlist.go

```diff
@@ -47,6 +47,15 @@ type PlatformSpec struct {
 	// OS specifies the operating system, for example `linux` or `windows`.
 	OS string `json:"os"`
 
+	// OSVersion is an optional field specifying the operating system
+	// version, for example `10.0.10586`.
+	OSVersion string `json:"osversion"`
```

Comment:
Let's mark this one `omitempty` too. There will be a lot of cases where OS version requirements are unspecified, such as most Linux images.